CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 Changelog since 1.4.1:
 ### Added
  - Add `Modern690Lc.downgrade_to_legacy` (unstable API)
+ - Added completion script for bash
```

Comment:
No longer in the right place since 1.4.2; needs to te in a new "[Unreleased]" block.

_(note: the context showed by GitHub around this change is misleading)_